### PR TITLE
Postmortem for malicious package publishes

### PR DESCRIPTION
**Pull request by hzoo**
Fixes https://github.com/eslint/eslint-scope/issues/39

### CODE REVIEW

File: _posts/2018-07-12-postmortem-for-malicious-package-publishes.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Postmortem for malicious package publishes
```
Comment (platinumazure) [inline _posts/2018-07-12-postmortem-for-malicious-package-publishes.md]: Wondering if a date should be included here?

File: _posts/2018-07-12-postmortem-for-malicious-package-publishes.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+> We, the ESLint team, are sorry for allowing this to happen. We hope that other package maintainers can learn from our mistakes and improve the security of the whole npm ecosystem.
+
+## Affected Packages
+
+* [`eslint-scope@3.7`](https://github.com/eslint/eslint-scope), a scope analysis library, is a dependency of several popular packages, including some older versions of [`eslint`](https://github.com/eslint/eslint) and the latest versions of [`babel-eslint`](https://github.com/babel/babel-eslint) and [`webpack`](https://github.com/webpack/webpack).
+* [`eslint-config-eslint@5.0`](https://github.com/eslint/eslint/tree/9aaf195ca691d307e8896096cefffe975218c701/packages/eslint-config-eslint) is a configuration used internally by the ESLint team, with very little usage elsewhere.
+
+## Attack Method
+
+Further details on the attack can be found [here]( https://gist.github.com/hzoo/51cb84afdc50b14bffa6c6dc49826b3e).
```
Comment (platinumazure) [inline _posts/2018-07-12-postmortem-for-malicious-package-publishes.md]: Does the leading space in front of the URL cause an issue?

File: _posts/2018-07-12-postmortem-for-malicious-package-publishes.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Postmortem for Malicious Packages Published on July 12th, 2018
```
Comment (btmills) [inline _posts/2018-07-12-postmortem-for-malicious-package-publishes.md]: Can you add frontmatter? Example: https://raw.githubusercontent.com/eslint/eslint.github.io/master/_posts/2018-07-08-eslint-v5.1.0-released.md

Tag it `security`?

File: _posts/2018-07-12-postmortem-for-malicious-package-publishes.md (hunk 2 above)
Comment (btmills) [reply _posts/2018-07-12-postmortem-for-malicious-package-publishes.md]: Appears to work fine

**Comment by denji** [comment]
https://github.com/evilsocket/opensnitch

**Pull request merged**